content/guides/hmr-react.md

```diff
@@ -212,25 +214,24 @@ export default App;
 ```
 
 The important thing to note in the code above is the `module` reference.
-First, we wrap the HMR code inside of `module.hot` check;
-webpack exposes `module` to the code, and if we are running with `hot: true` configured,
-we'll enter the inside of the conditional.
 
-While the module API offers more options than what's above, the most
-important element is the `module.hot.accept` call.
-It specific how to handle changes to specific dependencies.
+1. Webpack will expose `module.hot` to our code since we set `devServer: { hot: true }`;
+
+2. Thus we can use the `module.hot` hook to enable HMR for specific resources (Here's `App.js`). The most important API here is `module.hot.accept`, which specifies how to handle changes to specific dependencies.
+
+3. Note that because Webpack 2 has built-in support for ES2015 modules, you won't need to re-require your root component in `module.hot.accept`. To make this work, you need to change the Babel ES2015 preset in `.babelrc` to be:
```

Comment:
Note that disabling babel's module plugin is not only necessary for HMR; if you don't disable it you'll run into many other issues (for some simple installs it might work though).